content/docs/command-reference/exp/show.md

```diff
@@ -281,6 +281,18 @@ $ dvc exp show --only-changed --sort-by=roc_auc --sort-order desc
  ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
 ```
 
+<details>
```

Comment:
Another alternative place I considered was somewhere in https://dvc.org/doc/user-guide/experiment-management/comparing-experiments.

Reply:
Why not both? 😄 It's a very small addition, so I don't see much harm including it there also.

Reply:
Updated `comparing-experiments.md` as well! 

<img width="1312" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/f41a00b4-4fc9-4fbb-8b14-d0592057803e">